website/docs/docs/get-started/getting-started-dbt-core.md

```diff
@@ -5,13 +5,13 @@ description: "Connecting your warehouse to dbt Core using the CLI."
 sidebar_label: "Getting started"
 ---
 
-When you use dbt Core to work with dbt, you will be editing files locally using a code editor, and running projects using the dbt command line interface (dbt CLI). If you'd rather edit files and run projects using the web-based Integrated Development Environment (IDE), you should refer to [Getting set up in dbt Cloud](/docs/develop/getting-started/getting-set-up).
+When you use dbt Core to work with dbt, you will be editing files locally using a code editor, and running projects using the dbt command line interface (dbt CLI). If you'd rather edit files and run projects using the web-based Integrated Development Environment (IDE), you should refer to [Getting set up in dbt Cloud](/docs/get-started/getting-started/set-up-dbt-cloud).
 
 ## Prerequisites
 
 * To use the dbt CLI, it's important that you know some basics of the Terminal. In particular, you should understand `cd`, `ls` and `pwd` to navigate through the directory structure of your computer easily.
 * Install dbt Core using the [installation instructions](/dbt-cli/install/overview) for your operating system.
```

Comment:
[This](https://deploy-preview-2186--docs-getdbt-com.netlify.app/dbt-cli/install/overview) redirects to https://deploy-preview-2186--docs-getdbt-com.netlify.app/docs/develop/installation which 404s

Reply:
Good catch! I fixed the redirects in [this commit](https://github.com/dbt-labs/docs.getdbt.com/pull/2186/commits/df78dcde6b6af084c2c5fad03c844fd4fe635e7e)